### ISSUE TITLE
Missing mermaid arrowhead

### ISSUE THREAD

**Issue by kentdotn** [evilz/vscode-reveal#533]
Hi guys,

I've found a something odd with vscode-reveal and mermaid sequence diagram.
The following markdown contains two diagram pages and each diagram have two lines with arrowheads:
```markdown
---
theme: "white"
---

# TEST

---

## Sequence 1

<div class="mermaid">
sequenceDiagram
    Participant Alice
    Participant Bob
    Alice ->> Bob: Hi
    Bob -->> Alice: Hi
</div>

---

## Sequence 2

<div class="mermaid">
sequenceDiagram
    Participant Alice
    Participant Bob
    Alice ->> Bob: Hi
    Bob -->> Alice: Hi
</div>
```

For the first diagram, vscode-reveal gives us a correct output:
<img width="480" alt="スクリーンショット 2020-08-24 17 03 31" src="https://user-images.githubusercontent.com/5977528/91019339-ce755680-e62b-11ea-931e-705bf7a65a50.png">

but for the second one, the renderer doesn't render the arrowheads:
<img width="480" alt="スクリーンショット 2020-08-24 17 03 42" src="https://user-images.githubusercontent.com/5977528/91019410-e4831700-e62b-11ea-8daf-a15b914ac054.png">

Any ideas? Or am I missing something?

**Comment by kentdotn** [comment]
Additional information:

When you add some nested slides with diagrams under the first page, these ones have arrowheads correctly.

**Comment by kentdotn** [comment]
I use VSCode 1.48.1 and vscode-reveal v4.0.3.

**Comment by kentdotn** [comment]
One more info:
When I export the slides in PDF, the arrows appear in every slides.

**Comment by kentdotn** [comment]
@tokiedokie's fork seems to fix this issue already:
https://github.com/tokiedokie/reveal-markdown

**Comment by evilz** [comment]
Fixed in [4.1.0 ](https://github.com/evilz/vscode-reveal/releases/tag/4.1.0)
